adapters/mqtt-vertx-base/src/main/java/org/eclipse/hono/adapter/mqtt/AbstractVertxBasedMqttProtocolAdapter.java

```diff
@@ -358,7 +368,7 @@ private void handleConnectionRequestResult(final MqttEndpoint endpoint,
             LOG.debug("connection to unauthenticated device [clientId: {}] closed", endpoint.clientIdentifier());
             metrics.decrementUnauthenticatedMqttConnections();
         });
-        endpoint.publishHandler(message -> onPublishedMessage(new MqttContext(message, endpoint)));
+        endpoint.publishHandler(message -> handlePublishedMessage(new MqttContext(message, endpoint)));
```

Comment:
the "handlePublishedMessage" should be used in the Javadoc as well.

Reply:
well, *handlePublishedMessage* is a private method which doesn't show up in the JavaDocs anyway and its only responsibility is to start and end the corresponding span. It does indeed invoke *onPublishedMessage*, though, so IMHO the JavaDoc comment is still correct and reflects the information that is relevant to a user/adopter, i.e. it points out the method he can plug into. WDYT?